starter-code/styles/base.css

```diff
@@ -1,4 +1,6 @@
 /* TODO: Add a rule for images, so they never exceed their container's width. */
+@media screen and (min-width: 800px){
```

Comment:
I think you guys may have missed this todo. So I'll have to dock a point. Feel free to complete and resubmit.